tests/by-util/test_mv.rs

```diff
@@ -1243,6 +1243,31 @@ fn test_mv_info_self() {
         .stderr_contains("mv: cannot move 'dir2' to a subdirectory of itself, 'dir2/dir2'");
 }
 
+#[test]
+fn test_mv_arg_interactive_skipped() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    at.touch("a");
+    at.touch("b");
+    ucmd.args(&["-vi", "a", "b"])
+        .pipe_in("N\n")
+        .ignore_stdin_write_error()
+        .fails()
+        .stderr_is("mv: overwrite 'b'? ")
+        .stdout_is("skipped 'b'\n");
+}
+
+#[test]
+fn test_mv_arg_interactive_skipped_vin() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    at.touch("a");
+    at.touch("b");
+    ucmd.args(&["-vin", "a", "b"])
+        .pipe_in("y\n")
```

Comment:
Why do you use `pipe_in` in this test case? It seems irrelevant as the test also works without it.
```suggestion
```

Reply:
because you want to test the yes and no behavior, no ? :)

Reply:
Yes, of course :) However, in this specific case, there is no yes/no-behavior to test. It doesn't matter what you pipe in, the output will always be the same.